runtime/tests/jvmtitests/agent/version.c

```diff
@@ -67,7 +67,7 @@ getVersionName(agentEnv * agent_env, jint version)
 		case JVMTI_VERSION_11:
 			return versionNames[4];
 #if JAVA_SPEC_VERSION >= 15
-		case JVMTI_VERSION_15:
+		case JVMTI_VERSION:
```

Comment:
This also won't work with the current code - the requested version is placed in the JVMTI environment, so versions prior to the current JDK level won't work.

Reply:
Since this is just test code, I would suggest removing this entirely, and printing the hexadecimal version numbers in the two places that reference it.

Reply:
Currently the `version` for `getVersionName()` are `JVMTI_VERSION_1_X` hence the test passes.

To test earlier `JVMTI` version(s) w/o explicit defines within `jvmti.h`, a separated PR can add such tests.

Reply:
The fix here is trivial, so I'd like to see it as part of this PR. There's no reason for added tests until we determine if the minor version matters.

Reply:
As per offline conversation, updated the test to print out JVMTI version in hex format, also removed `getVersionName()`.